macstadium-shared-1/main.tf

```diff
@@ -440,6 +440,16 @@ module "vsphere_janitor_custom_5" {
   index = "${var.index}"
 }
 
+module "vsphere_monitor" {
+  source = "../modules/vsphere_monitor"
+  host_id = "${module.macstadium_infrastructure.util_uuid}"
+  ssh_host = "${module.macstadium_infrastructure.util_ip}"
+  ssh_user = "${var.ssh_user}"
+  version = "${var.vsphere_monitor_version}"
```

Comment:
This variable isn't defined yet, waiting on https://github.com/travis-ci/vsphere-monitor/issues/7 to get fixed so I have a version to set.